PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -184,8 +192,12 @@ internal class PayPalInternalClient(
             ?: redirectUri.getQueryParameter("token")
     }
 
+    private fun PayPalRequest.isBillingAgreement(): Boolean = this is PayPalVaultRequest
+
     companion object {
         private const val CREATE_SINGLE_PAYMENT_ENDPOINT = "paypal_hermes/create_payment_resource"
         private const val SETUP_BILLING_AGREEMENT_ENDPOINT = "paypal_hermes/setup_billing_agreement"
+        private const val VAULT_FLOW_TYPE = "va"
+        private const val ECS_FLOW_TYPE = "ecs"
```

Comment:
```suggestion
        private const val SINGLE_PAYMENT_FLOW_TYPE = "ecs"
```

`ECS` might be a little confusing since the acronym doesn't currently exist in the codebase.

Reply:
An astute observation.